easy-connect.h

```diff
@@ -174,8 +202,22 @@ NetworkInterface* easy_connect(bool log_messages = false) {
         }
         return NULL;
     }
+
+#if MBED_CONF_APP_NETWORK_INTERFACE != CELLULAR
+    const char *mac_addr = network_interface->get_mac_address();
```

Comment:
Or just use the print_MAC() function here as well? It already has the ifdef protection for  cellular, no need tor repeat it twice.

Reply:
`print_MAC()` has already been called a few lines above, in which case do we need lines 206 to 215, and lines 218 to 220, at all?

Reply:
Well, if it already gets printed for other stacks - no, no need to add it again. :-)
Testing needed.